ui/src/CollectionExerciseDetails.js

```diff
@@ -175,22 +175,14 @@ class CollectionExerciseDetails extends Component {
       return;
     }
 
-    let printTemplate = null;
-    if (this.state.newActionRuleType === "PRINT") {
-      printTemplate = "printTemplates/" + this.state.newActionRulePackCode;
-    }
-
     const newActionRule = {
-      id: uuidv4(),
       type: this.state.newActionRuleType,
       triggerDateTime: new Date(
         this.state.newActionRuleTriggerDate
       ).toISOString(),
-      hasTriggered: false,
       classifiers: this.state.newActionRuleClassifiers,
-      printTemplate: printTemplate,
```

Comment:
you can't just remove this... you've broken print action rules

Reply:
Does it need to be there? He's passing in the pack code and gets the print template from the DB from the packcode then.

Reply:
my bad. can remove